site/operational-guide/interactive.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Interactive Deployment
+menu_order: 20
+---
+
+This pattern is recommended for exploration and evaluation only, as
+the commands described herein are interactive and not readily amenable
+to automation and configuration management. Nevertheless, the
+resulting weave network will survive host reboots without the use of a
+systemd unit as long as Docker is configured to start on boot.
+
+### Bootstrap
+
+On initial peer:
+
+    weave launch
+
+### Add a Peer
+
+On new peer:
+
+    weave launch <extant peers>
+
+Where `<extant peers>` means all peers in the network, initial and
+subsequently added, which have not been explicitly removed. It should
+include peers which are temporarily offline or stopped.
+
+You must then execute:
+
+    weave prime
+
+to ensure that the new peer has joined to the existing network; you
+_must_ wait for this to complete successfully before moving on to add
+further new peers. If this command blocks it means that there is some
+issue (such as a network partition or failed peers) that is preventing
+a quorum from being reached - you will need to [address
+that](/site/troubleshooting.md) before moving on.
+
+Then optionally on each existing peer:
+
+    weave connect <new peer>
+
+This step is not mandatory, but improves the robustness of network
+reformation in the face of node failures as _nodes do not remember to
+connect to discovered peers on restart_.
```

Comment:
I'm uneasy about this part.  Is it really needed in the case where you have given each new peer all the previous peers?